features/contain-size.yml

```diff
@@ -0,0 +1,6 @@
+name: Size containment
+description: "The `contain: size` CSS declaration TODO."
```

Comment:
```suggestion
description: "The `contain: size` CSS declaration sets an element's dimensions exclusively by its `height` and `width` properties, ignoring its contents and descendants. This permits the browser to avoid slower layout calculations."
```